toolkit/tools/imagegen/configuration/packagerepo.go

```diff
@@ -139,15 +165,13 @@ func (p *PackageRepo) repoUrlIsValid() (err error) {
 
 func writeAdditionalFields(stringBuilder *strings.Builder) (err error) {
 	const (
-		gpgKey       = "gpgkey=file:///etc/pki/rpm-gpg/MICROSOFT-RPM-GPG-KEY file:///etc/pki/rpm-gpg/MICROSOFT-METADATA-GPG-KEY\n"
 		enable       = "enabled=1\n"
-		gpgCheck     = "gpgcheck=1\n"
 		repogpgCheck = "repo_gpgcheck=1\n"
```

Comment:
Shouldn't the repogpgCheck field also be customizable? If this field is set to true, then yum will perform a signature check on the repodata and it will fail if there's no repo gpg key.

Reply:
Ah, missed that. Will also include it.